### PR TITLE
Add Metabase Add-On

### PR DESCRIPTION
**Pull request by dsferruzza**
## Describe your PR

This adds a new page in the documentation for the upcoming Metabase add-on.

## Checklist

- [x] ~~My PR is related to an opened issue~~
- [x] I've read the [contributing guidelines](/CleverCloud/documentation/blob/main/CONTRIBUTING.md)

## Reviewers

@davlgd

### CODE REVIEW

File: content/doc/addons/metabase.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+Refer to the [Clever Tools documentation](https://github.com/CleverCloud/clever-tools/blob/master/docs/addons-backups.md) for more details on add-on management.
+
+## Acessing the Metabase Interface
+
+Once deployed, Metabase is accessible through an URL provided in the add-on's `Service dependencies` page. You can also go in the environment variables page of the Java application and look for `MB_SITE_URL` value. To get this URL with Clever Tools, use the add-on ID:
+
+```
+$ clever addon env ADDON_ID
+METABASE_URL="https://an_id-metabase.services.clever-cloud.com"
```
Comment (dsferruzza) [inline content/doc/addons/metabase.md:64]: Make sure that this URL makes sense (cc @allemas)

File: content/doc/addons/metabase.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+## Plan sizing
+
+By default, Metabase on Clever Cloud uses small-size resources, i.e:
+
+- XS Java
+- XXS Small Space PostgreSQL
+
+They are dimensioned to suit a majority of needs. You can however manage and adjust them directly [in the Console](https://console.clever-cloud.com/) to fit your needs. You can for example change their settings, migrate to a larger storage database, etc. Vertical auto-scalability is available for this service. Different plans for Java and PostgreSQL are available on Clever Cloud.
+
+Metabase default resources are dimensioned to suit low concurrent usage of Metabase. When you have multiple users loading large dashboards concurrently, if your instance experiences crashes due to Out of Memory (OOM) issues, you should use a larger flavor for the Java application.
```
Comment (dsferruzza) [inline content/doc/addons/metabase.md:87]: Maybe this paragraph should replace/be merged with the previous one? It feels like a partial repetition... (cc @davlgd)

File: content/doc/addons/metabase.md (hunk 1 above)
Comment (davlgd) [reply content/doc/addons/metabase.md:64]: It's the env var as sent back by the add-on once created. As we planned to modify it, this change LGTM

**Review by davlgd — APPROVED** [review]
LGTM, thanks for this!

**Comment by maximepvrt** [comment]
Great job @dsferruzza !

**Pull request merged**